tests/integrations/test_dsm.py

```diff
@@ -5,7 +5,10 @@
 from utils import weblog, interfaces, context, bug, missing_feature, scenarios
 
 
-@missing_feature(condition=context.library != "java", reason="Full Kafka instrumentation only on Java")
+@missing_feature(
```

Comment:
We usually express this with the `@released` decorator at class level. For example:
```
@released(python="?", cpp="?", golang="?", java="1.12.0", dotnet="?", nodejs="?", php="?", ruby="?")
```